.travis/release.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 MESSAGE=$(git log --format=%B -n 1 $TRAVIS_COMMIT)
-git clone ${REPO}.git
+git clone ${REPO}.git -b $1
 cd public
 cp ../${REPO_DIR}/Jenkinsfile ./Jenkinsfile
```

Comment:
This is actually unnecessary for all branches except for prod-stable and prod-beta but I actually copied Jenkinsfiles into all of the branches anyway so it won't hurt anything.